src/Doctrine/ORM/PixelRepository.php

```diff
@@ -4,20 +4,28 @@
 
 namespace Setono\SyliusFacebookPlugin\Doctrine\ORM;
 
+use Doctrine\DBAL\Types\Types;
+use Setono\SyliusFacebookPlugin\Model\PixelInterface;
 use Setono\SyliusFacebookPlugin\Repository\PixelRepositoryInterface;
 use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
 use Sylius\Component\Channel\Model\ChannelInterface;
+use Webmozart\Assert\Assert;
 
 class PixelRepository extends EntityRepository implements PixelRepositoryInterface
 {
     public function findEnabledByChannel(ChannelInterface $channel): array
     {
-        return $this->createQueryBuilder('o')
+        $result = $this->createQueryBuilder('o')
             ->andWhere(':channel MEMBER OF o.channels')
             ->andWhere('o.enabled = true')
-            ->setParameter('channel', $channel)
+            ->setParameter('channel', $channel, Types::OBJECT)
```

Comment:
The warning that caused you to add `Types::OBJECT` is one I usually suppress because I am not even sure what implications it has to define it (if I define it wrong)

Reply:
You're right. This particular line broke things. 
However, as an example, passing `Types::STRING` for strings works well.

Reply:
but I think you should just suppress it, so we don't accidentally break things :D

Reply:
I suppressed at this particular line.
Or you mean globally?